socketDataHandler.js

```diff
@@ -40,6 +41,11 @@ class SocketDataHandler {
       this.socket = socket
       this.keepAliveTimeout = []
       this.remoteServer = null
+      this.player = {
```

Comment:
We need to keep track of a players position for the routing so that we know what server should get the packets